libparsec/Cargo.toml

```diff
@@ -22,6 +22,7 @@ test-utils = [
     "libparsec_platform_ipc/test-with-testbed",
     "libparsec_types/test-mock-time",
 ]
+mountpoint = ["dep:libparsec_platform_mountpoint"]
```

Comment:
What is the need here ?
Given this feature is used in all builds, and the build is broken when not enabling it (see my other comment) I think we should just get rid of it

Reply:
The PR description state:

> `libparsec` as a new feature `mountpoint` that provide mountpoint. That
simplify compilation of static cli since we do not have to include
`libfuse`.

Reply:
There is no simplification here since we add code to support a new feature, feature which is always enabled.

On top of that we want to have fuse support in the CLI, and this PR was able to do that fine ;-)

So I'd rather not have this feature entirely to stay KISS

Reply:
reverted